### PR TITLE
Add Nim 0.20.2 and 0.20.0

### PR DESCRIPTION
**Pull request by genotrance**

### CODE REVIEW

File: docsets/Nim/docset.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "name": "Nim",
+    "version": "0.20.2",
+    "archive": "Nim.tgz",
```
Comment (Kapeli) [inline docsets/Nim/docset.json]: You need to add this archive as well.

Comment (genotrance) [reply docsets/Nim/docset.json]: Done

**Comment by Kapeli** [comment]
Thank you! 👍

**Comment by genotrance** [comment]
When will this get picked up? The tgz files are still in the repo and no feeds have been generated yet.

**Comment by Kapeli** [comment]
What do you mean? The archives have been processed and no longer in the repo - https://github.com/Kapeli/Dash-User-Contributions/tree/master/docsets/Nim and the docset is installable from within Dash.

**Pull request merged**